packages/inter-protocol/src/vaultFactory/storeUtils.js

```diff
@@ -128,15 +128,22 @@ harden(fromVaultKey);
  *
  * @param {PriceQuote} quote
  * @param {Ratio} compoundedInterest
+ * @param {Ratio} margin
  * @returns {number}
  */
-export const normalizedCollRatio = (quote, compoundedInterest) => {
+export const normalizedCollRatio = (quote, compoundedInterest, margin) => {
```

Comment:
I would like a comment that explains what this is supposed to do rather than who uses it.